apistructs/pipeline.go

```diff
@@ -342,8 +347,42 @@ type PipelinePageListRequest struct {
 
 	// internal use
 	SelectCols []string `schema:"-" ` // 需要赋值的字段列表，若不声明，则全赋值
-	AscCols    []string `schema:"-"`
-	DescCols   []string `schema:"-"`
+	AscCols    []string `schema:"ascCol"`
+	DescCols   []string `schema:"descCol"`
+
+	// pipeline definition search
+	PipelineDefinitionRequest           *PipelineDefinitionRequest
+	PipelineDefinitionRequestJSONBase64 string `schema:"pipelineDefinition"`
```

Comment:
adjust schema key

Reply:
For the time being, there is no good scheme for receiving values. All fields of definition can be moved out. If necessary, they can be done later. Let's shelve it first